config/initializers/mojfile_uploader_client.rb

```diff
@@ -0,0 +1,6 @@
+require 'mojfile_uploader_api_client'
+
+MojFileUploaderApiClient::HttpClient.configure do |client|
+  client.base_url = 'http://localhost:9292'
```

Comment:
Shouldn't this be coming from an ENV var? We can default it using ENV.fetch(VAR, 'http...')